File: tank_cilindrical.m
function dh_dt = tank_cilindrical(t,y,A,Qin,Cd)
    % calculate derivative of the Level
    
    g = 9.81;
    R = 0.042; % Raio em metros
    
    b = A*Cd*sqrt(2*g);
    
    k1 = 1/(pi*(R)^2);
    k2 = b/(pi*(R)^2);

    dh_dt = Qin*k1 - k2*y^(1/2);
end

File: tank_cilindrical.py

def dh_dt(t,y,A,Qin,Cd):
    import math
    # calculate derivative of the Level
    
    g = 9.81;
    R = 0.042; % Radio meters
    
    b = A*Cd*math.sqrt(2*g);
    
    k1 = 1/(math.pi*(R)^2);
    k2 = b/(math.pi*(R)^2);


    dh_dt = Qin*k1 - k2*y**(1/2);

    return dh_dt